components/fieldlabel/index.css

```diff
@@ -66,27 +81,29 @@ governing permissions and limitations under the License.
 
 .spectrum-FieldLabel--left {
   display: inline-block;
-  padding-block: var(--spectrum-fieldlabel-m-side-padding-top) 0;
-  padding-inline: 0 var(--spectrum-fieldlabel-m-side-padding-right);
+  padding-block: var(--spectrum-fieldlabel-side-padding-top) 0;
+  padding-inline: 0 var(--spectrum-fieldlabel-side-padding-right);
 
   & .spectrum-FieldLabel-requiredIcon {
-    margin-block: var(--spectrum-fieldlabel-m-side-asterisk-margin-y) 0;
-    margin-inline: var(--spectrum-fieldlabel-asterisk-gap) 0;
+    margin-block: var(--spectrum-field-label-text-to-asterisk) 0;
+    margin-inline: var(--spectrum-fieldlabel-asterisk-gap) 0; 
   }
 }
 
 .spectrum-FieldLabel--right {
   display: inline-block;
   text-align: end;
-  padding-block: var(--spectrum-fieldlabel-m-side-padding-top) 0;
-  /* todo: correct missing DNA variable */
-  padding-inline: 0 var(--spectrum-fieldlabel-m-side-padding-right);
+  padding-block: var(--spectrum-fieldlabel-side-padding-top) 0; 
+  padding-inline: 0 var(--spectrum-fieldlabel-side-padding-right);
+
 }
 
 .spectrum-Form {
-  --spectrum-tableform-border-spacing: 0 var(--spectrum-global-dimension-size-300);
-  --spectrum-tableform-margin: calc(var(--spectrum-global-dimension-size-250) * -1) 0;
-  --spectrum-tableform-labelsabove-item-gap: var(--spectrum-global-dimension-size-100);
+  /* for /docs/form.html to set field-label inside form */
+  --spectrum-tableform-border-spacing: 0 var(--spectrum-spacing-300);
+  /* matching 20px, missing global token for 20px */
```

Comment:
@pfulton  We are missing 20px from the core-tokens. Is it ok to get 20px with a core-tokens addition?

Reply:
Yeah, I think so. This will be something we'll need to keep an eye on, as I think I'd much rather have it coming as a definition from core tokens, but in the meantime (or if we have to), let's do this for now with an eye on possibly defining it ourselves in the `custom-*` files at some future point.

Reply:
If we're missing 20px in spacing values, then perhaps the component was designed wrong (or core tokens needs to have a new spacing value). We shouldn't be calcing this.